src/features/auth/authSelectors.ts

```diff
@@ -49,8 +49,11 @@ export const instanceSelector = createSelector([handleSelector], (handle) => {
   return getInstanceFromHandle(handle);
 });
 
-export const urlSelector = (state: RootState) =>
-  instanceSelector(state) ?? state.auth.connectedInstance;
+export const urlSelector = (state: RootState) => {
+  if (import.meta.env.VITE__TEST_MODE) return state.auth.connectedInstance;
```

Comment:
curious: shouldn't this be implemented by mocking when running the tests? :thinking:

Reply:
That would be a nice improvement

Reply:
Sorry for taking this long to respond.

Not entirely clear on what we would want to mock here, @sharunkumar. Could you please expand on it? Feel free to chime in, @aeharding.